src/ncurl.act

```diff
@@ -290,7 +290,16 @@ actor CmdGetSchema(env, args, log_handler):
                     else:
                         filename = "{filename}.yang"
 
-                    filepath = "{output_dir}/{filename}"
+                    if output_dir not in {"", "."}:
+                        fs = file.FS(file.FileCap(env.auth))
+                        try:
+                            await async fs.mkdir(output_dir)
+                        except OSError as err:
+                            if "already exists" not in err.error_message:
```

Comment:
Bah, I am so annoyed by those errors. I have the more declarative view that mkdir means "I want this directory to exist" rather than the imperative "you must create this directory" - and from the more declarative view, it clearly is not an error if the directory already exist. I thought I fixed this already but maybe I'm misremembering or possibly I mixed it up with some remove command

Reply:
https://github.com/actonlang/acton/issues/2436